sonobuoy-conformance.yaml

```diff
@@ -52,7 +52,7 @@ metadata:
 data:
   config.json: |
     {
-        "Description": "CNCF v1.7 or v1.8 Conformance Results",
+        "Description": "CNCF v1.7, v1.8, or v1.9 Conformance Results",
```

Comment:
```
"Description": "CNCF v1.8 or v1.9 Conformance Results",
```

Now that K8s 1.9 has been released, new certifications of 1.7 are no longer excepted. See https://github.com/cncf/k8s-conformance/blob/master/faq.md#what-versions-of-kubernetes-can-be-certified.

Reply:
Comment change otherwise then lgtm from me.

Reply:
Ty
This has been addressed. :+1: